chromadb/auth/__init__.py

```diff
@@ -35,13 +41,39 @@ class AuthInfoType(Enum):
     METADATA = "metadata"  # gRPC
 
 
+class UserIdentity(EnforceOverrides, ABC):
```

Comment:
we may want the identity to also give the database, in the hosted service case, we may want api key to fully quality the db. This is additive though so can add later

Reply:
already did that

Reply:
i see now - was on old